src/patternfly/components/Table/table-grid.scss

```diff
@@ -139,7 +139,7 @@
   border: none;
 
   // reset cell modifications
-  tr:where(.#{$table}__tr) > * {
+  tr:where(.#{$table}__tr) > :where(:is(th, td)) {
```

Comment:
Can you just do this as `tr:where(.#{$table}__tr) > :where(th, td)` ?
Or even `:where(.#{$table}__tr) > :where(th, td)`?